toml/_parser.ts

```diff
@@ -153,7 +156,7 @@ export function deepAssignWithTable(target: Record<string, unknown>, table: {
   value: Record<string, unknown>;
 }) {
   if (table.key.length === 0 || table.key[0] == null) {
-    throw new Error("Unexpected key length");
+    throw new Error("Key length must be a positive number");
```

Comment:
```suggestion
    throw new Error("Cannot parse the TOML: Key length must be a positive number");
```

Reply:
Shouldn't it be `Key length is not a positive number`?

Reply:
```suggestion
    throw new Error("Cannot parse the TOML: key length is not a positive number"));
```